docs/sparrow_run.md

```diff
@@ -37,3 +37,4 @@ sparrow run [flags]
 
 * [sparrow](sparrow.md)	 - Sparrow, the infrastructure monitoring agent
 
+###### Auto generated by spf13/cobra on 10-Jan-2024
```

Comment:
Disable this auto gen tag since the pre-commit hook would always fail once a day.